packages/credentials/src/types.ts

```diff
@@ -0,0 +1,79 @@
+
+/**
+ * @see {@link https://www.w3.org/TR/vc-data-model/#credentials | VC data model}
+ */
+export type VerifiableCredential = {
+  /** see {@link https://www.w3.org/TR/vc-data-model/#contexts | Contexts} */
+  '@context': ['https://www.w3.org/2018/credentials/v1', ...string[]];
+  credentialSubject: CredentialSubject | CredentialSubject[];
+  /** @see {@link https://www.w3.org/TR/vc-data-model/#identifiers | Identifiers} */
+  id?: string;
+  /** @see {@link IssuerType} */
+  issuer: Issuer;
+  /** @see {@link https://www.w3.org/TR/vc-data-model/#types | Types} */
+  type?: string[] | string;
+  /** @see {@link https://www.w3.org/TR/vc-data-model/#issuance-date | Issuance Date} */
+  issuanceDate: string;
+  /** @see {@link https://www.w3.org/TR/vc-data-model/#expiration | Expiration} */
+  expirationDate?: string;
+  /** @see {@link CredentialStatusReference} */
+  credentialStatus?: CredentialStatusReference;
+  [x: string]: any;
```

Comment:
Tiny nit -- apologies for raising 🤓 

`[x: string]` convention is used throughout Veramo's lib but we've typically been using `[key: string]` or `[prop: string]` in Web5 SDK / DWN SDK.  Although to be fair, we've not been consistent.